### ISSUE TITLE
Enhancement: integrate Fauxbar with the vomnibox

### ISSUE THREAD

**Issue by drewfrank** [philc/vimium#530]
The [Fauxbar](http://code.google.com/p/fauxbar/) extension provides alternate address box and search box widgets with vastly improved completion and suggestion capabilities compared to Chrome's default "omnibox." It would be wonderful to combine this functionality with the new vomnibox widget of Vimium. This would have two benefits:
1.  Give Vimium users access to the very nice functionality of Fauxbar. Without integration, Vimium users will have to either disable the vomnibox or miss out on parts of Fauxbar.
2.  Give Fauxbar users a nicer way to access its functionality. Fauxbar provides its own fancy address and search boxes, but displays them on a separate "plugin page" instead of overlaying them like the vomnibox. This means that whenever a user pushes Ctrl+L or Ctrl+K they navigate away from their current page to the Fauxbar page, which clutters browser history and is generally undesirable. Incidentally, since Vimium provides an easy way to unmap all keybindings I can imagine Fauxbar users installing Vimium _just_ for the vomnibox, even if they are not Vim users / keyboard gurus.

In summary: if the Fauxbar backend could be wired up to the Vomnibox frontend it would be a big improvement for both Vimium and Fauxbar and probably result in many happy users :).
